.jenkins_nightly

```diff
@@ -80,11 +80,12 @@ pipeline {
                     steps {
                         sh '''
                           DEBIAN_FRONTEND=noninteractive && \
-                          apt-get update && apt-get upgrade -y && apt-get install -y \
-                          cmake \
-                          && \
+                          apt-get update && apt-get upgrade -y && \
```

Comment:
Why do you need to do any of these if you are not installing anything?

Reply:
This worked. We can try without it.

Reply:
Seems to be fine without it. Updated.

Reply:
You run everything wight not just building the image?

Reply:
Yes, I configured the same way we specify here, build and ran the tests.